RELEASE/scripts/autoscend/auto_restore.ash

```diff
@@ -1797,6 +1797,12 @@ boolean acquireHP(int goal, int meat_reserve, boolean useFreeRests)
 		return false;
 	}
 	
+	// HP is irrelevant in Pocket Familiars, this removes the function to restore HP
```

Comment:
there is one time it matters. which is when doing the fantasy realm bandits for a fat loot token. kol devs said it is using normal combat there by design (it does match the joke)
so those are 5 battles where you need to win via "attack with weapon" and combat items

Reply:
Fantasyrealm is no longer accessible in pokefam due to being out of standard

Reply:
> Fantasyrealm is no longer accessible in pokefam due to being out of standard

oooh... because pokefam was made into "forever in standard"... right.
one of the worst things about it is that it never went out of standard because they never really finishing coding it.